projects/storyteller/webapp/src/app/components/header/header.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, OnInit,Output, EventEmitter } from '@angular/core';
+
+@Component({
+  selector: 'app-header',
+  templateUrl: './header.component.html',
+  styleUrls: ['./header.component.scss']
+})
+export class HeaderComponent implements OnInit {
+  @Output() public sidenavToggle = new EventEmitter();
+  constructor() { }
+
+  public onToggleSidenav = () => {
```

Comment:
No need to set `public`, because all variables and methods are public by default.

Reply:
You have to set it only with injected services. Example:
```typescript
constructor(
  public firebaseService: FirebaseService,
  private encodingService: EncodingService,
) { }
```